integration/util.go

```diff
@@ -140,7 +125,7 @@ func CopyFileToSharedDir(s *e2e.Scenario, src, dst string) error {
 	return WriteFileToSharedDir(s, dst, content)
 }
 
-func tempoBackoff() util.BackoffConfig {
+func TempoBackoff() util.BackoffConfig {
```

Comment:
Is there a reason you made this function public?

Reply:
Yeah this is used in the backend package within `e2e/`
https://github.com/grafana/tempo/pull/883/files#diff-59d063e3550f93433d370fd02888b5f23242d675d1b2435e771136d3638776caR107

Reply:
Oh okay, missed that while checking if it's used